### ISSUE TITLE
Python 3 breaks SEQ test

### ISSUE THREAD

**Issue by jhkennedy** [E3SM-Project/E3SM#2940]
Running the `SEQ.f19_g16.X` test on `cori-knl_intel` with python 3 (but not python 2) results in a SHAREDLIB_BUILD error: 

```
Finished SHAREDLIB_BUILD for test SEQ.f19_g16.X.cori-knl_intel in 12.735906 seconds (FAIL). [COMPLETED 1 of 1]
    Errors were:
        "Building test for SEQ in directory....
        .../env_mach_specific.xml already exists, delete to replace
        WARNING: Test case setup failed. Case2 has been removed, but the main case may be in an inconsistent state. If you want to rerun this test, you should create a new test rather than trying to rerun this one.
        ERROR: Wrong type for entry id 'NTASKS'"
```

--- 
Note: This was 1/2 issues originally reported in #2935 , but moved here to separate the issues.

**Comment by jhkennedy** [comment]
This is caused by the change in the division operator in python 3 from truncated division to true division. The fix is simple; use the truncated division operator (also available in python 2 since 2.2). 

I'll fix this in ESMCI/CIME, but if this is holding up anyone's workflow, you can apply this patch:

```diff
diff --git a/cime/scripts/lib/CIME/SystemTests/seq.py b/cime/scripts/lib/CIME/SystemTests/seq.py
index ed3ec92570..716215db56 100644
--- a/cime/scripts/lib/CIME/SystemTests/seq.py
+++ b/cime/scripts/lib/CIME/SystemTests/seq.py
@@ -32,7 +32,7 @@ class SEQ(SystemTestsCompareTwo):
                 self._case.set_value("ROOTPE_{}".format(comp), 0)
         else:
             totalpes = self._case.get_value("TOTALPES")
-            newntasks = max(1, totalpes/len(comp_classes))
+            newntasks = max(1, totalpes//len(comp_classes))
             rootpe = newntasks
 
             for comp in comp_classes:
```

**Comment by jhkennedy** [comment]
Will be fixed in CIME by ESMCI/cime#3118.

**Comment by jhkennedy** [comment]
Now on master.
